web/src/main/java/com/epam/esm/config/DataBaseConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.esm.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ComponentScan("com.epam.esm")
+@EnableWebMvc
+@PropertySource("classpath:application.properties")
+public class DataBaseConfig {
+    @Value("${dev.datasource.driver}")
+    private String driverClassName;
+
+    @Value("${dev.datasource.url}")
+    private String url;
+
+    @Value("${dev.datasource.username}")
+    private String username;
+
+    @Value("${dev.datasource.password}")
+    private String password;
+
+    @Bean
+    public DataSource dataSource() {
+        DriverManagerDataSource dataSource = new DriverManagerDataSource();
```

Comment:
This class is not an actual connection pool.
https://docs.spring.io/spring-framework/docs/2.0.x/javadoc-api/org/springframework/jdbc/datasource/DriverManagerDataSource.html
Please, use a connection pool, for example Hikari

Reply:
Fixed